webrtc.html

```diff
@@ -2099,7 +2140,8 @@
 
           <p>When the <dfn id=
           "dom-peerconnection-createdatachannel"><code>createDataChannel()</code></dfn>
-          method is invoked, the user agent MUST run the following steps.</p>
+          method is invoked, the user agent MUST <a>queue an RTCPeerConnection
+          task</a> to run the following steps:</p>
```

Comment:
We can't queue a task here since createDataChannel() returns a DataChannel object.